src/components/components.scss

```
@@ -12,4 +12,8 @@
   'Navbar/Navbar',
   'Sidebar/Sidebar',
   'TextArea/TextArea',
-  'Hero/Hero';
+  'Hero/Hero',
+  'TitleBody/TitleBody',
+  'Image/ImageBanner/ImageBanner',
+  'Title/TitleSubtitleBody/TitleSubtitleBody',
+  'Image/ImageTitleBodyButton/ImageTitleBodyButton';
```

these are too nested lets pull them out and just put in the components directory as: 

```suggestion
  'ImageBanner/ImageBanner',
  'TitleSubtitleBody/TitleSubtitleBody',
  'ImageTitleBodyButton/ImageTitleBodyButton';
```